internal/ent/hooks/usersettings.go

```diff
@@ -28,6 +29,21 @@ func HookUserSetting() ent.Hook {
 				return nil, rout.InvalidField(rout.ErrOrganizationNotFound)
 			}
 
+			if m.Op().Is(ent.OpUpdateOne) {
+				userID, err := auth.GetSubjectIDFromContext(ctx)
+				if err == nil && userID != "" {
+					// if webauthn is disabled, clean up the passkey we stored previously
+					if allowed, _ := m.IsWebauthnAllowed(); !allowed {
+						_, err := m.Client().Webauthn.Delete().
+							Where(webauthn.OwnerID(userID)).
+							Exec(ctx)
+						if err != nil {
+							return nil, err
+						}
+					}
+				}
```

Comment:
this _shouldnt_ ever happen here since its an update to user settings, but we are currently silently ignoring the error if it did. 
```suggestion
				if err != nil {
					return err
				}
				
				if userID != "" {
					// if webauthn is disabled, clean up the passkey we stored previously
					if allowed, _ := m.IsWebauthnAllowed(); !allowed {
						_, err := m.Client().Webauthn.Delete().
							Where(webauthn.OwnerID(userID)).
							Exec(ctx)
						if err != nil {
							return nil, err
						}
					}
				}
```

Reply:
ahhh gotcha. thats actually a good point - just add a comment about why we are ignoring it then